R/blpAuthenticate.R

```diff
@@ -64,22 +66,30 @@ blpAuthenticate <- function(uuid=getOption("blpUUID", NULL),
                             host=getOption("blpLoginHostname", "localhost"),
                             ip.address=getOption("blpLoginIP", NULL),
                             con=defaultConnection(),
-                            default=TRUE) {
+                            default=TRUE,
+                            isEmrsId = getOption("isEmrsId", FALSE)) {
     if(is.null(uuid)) {
         ## no UUID, assume B-PIPE or SAPI with application ID
-        blpAuth <- authenticate_Impl(con, NULL, NULL)
+        blpAuth <- authenticate_Impl(con, NULL, NULL, FALSE)
+    } else if(isEmrsId) {
+        if(is.null(uuid) || is.null(ip.address)){
+            stop("when isEmrsId is TRUE, uuid and ip.address must be passed")
```

Comment:
Nit: Maybe capitalize 'When' and call `, call. = FALSE` ?

Reply:
will do

Reply:
I capitalized the "W", but was a bit confused by the second suggestion - happy to do it, just want to clarify what you mean by "call."